lighthouse-core/gather/gatherers/trace-elements.js

```diff
@@ -49,12 +54,70 @@ function collectTraceElements(attributeMarker) {
 
 class TraceElements extends Gatherer {
   /**
-   * @param {LH.TraceEvent | undefined} lcpEvent
+   * @param {LH.TraceEvent | undefined} event
    * @return {number | undefined}
    */
-  static getNodeIDFromTraceEvent(lcpEvent) {
-    return lcpEvent && lcpEvent.args &&
-      lcpEvent.args.data && lcpEvent.args.data.nodeId;
+  static getNodeIDFromTraceEvent(event) {
+    return event && event.args &&
+      event.args.data && event.args.data.nodeId;
+  }
+
+  /**
+   * @param {Array<number>} rect
```

Comment:
`number[]` please for simple types :)

to be fair, summoning @brendankenny

Reply:
> `number[]` please for simple types :)

> to be fair, summoning @brendankenny

Hard disagree :P

Since we were unable to agree on this in the past I've been leaving it at author's discretion (modulo changing existing code by another author modulo context of the change, etc), which is our usual mode for disagreement (see prettier's terrible `?` placement :)